dataactbroker/handlers/fileHandler.py

```diff
@@ -1,11 +1,14 @@
 import os
+import boto
```

Comment:
I just used the one that we use in the most places. Will it continue working with everything or will we have to run another test if I update?